templates/blogpost.md.ejs

```diff
@@ -27,6 +27,31 @@ function outputList(items) {
 
 %>
 
+<% if (prereleaseMajorVersion) { %>
```

Comment:
Am I not understanding this change correctly, or based on the changes in Makefile, `prereleaseMajorVersion` will always have some sort of value?

Reply:
You're absolutely correct, I forgot to compare the versions in the Makefile and only hydrate the context object when we're in a pre-release. I'll need to fix that. Thanks!